xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -368,10 +395,21 @@ The JSON token value represents a number less than <see cref="F:System.Byte.MinV
       <Parameters />
       <Docs>
         <summary>Reads the next JSON token value from the source and parses it to a <see cref="T:System.DateTime" />.</summary>
-        <returns>The date and time, if the entire UTF-8 encoded token value can be successfully parsed.</returns>
-        <remarks>This method only creates a DateTime representation of JSON strings that conform to the ISO 8601-1 extended format (see [DateTime and DateTimeOffset support in System.Text.Json](~/docs/standard/datetime/system-text-json-support.md)).</remarks>
+        <returns>The date and time offset, if the entire UTF-8 encoded token value can be successfully parsed.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method only creates a DateTime representation of JSON strings that conform to the ISO 8601-1 extended format (see [DateTime and DateTimeOffset support in System.Text.Json](~/docs/standard/datetime/system-text-json-support.md)).
```

Comment:
@ahsonkhan is DateTime here supposed to be treated as a "JSON DateTime" as opposed to a "C# System.DateTime instance"? Similar to the JSON boolean?

If that's the case, then we could make it bold, like we've been doing with the JSON booleans:

```suggestion
This method only creates a **DateTime** representation of JSON strings that conform to the ISO 8601-1 extended format (see [DateTime and DateTimeOffset support in System.Text.Json](~/docs/standard/datetime/system-text-json-support.md)).
```

If I'm wrong, feel free to ignore my suggestion.

Reply:
Same question applies to DateTimeOffset below. I suspect we are using C# objects, not JSON objects, because that's the returned value by the methods.

Reply:
Right, JSON doesn't have a concept of DateTime/DateTimeOffset. These are referring to C# objects, unlike JSON boolean).